### PR TITLE
Curriculum redo

### PR DESCRIPTION
**Pull request by jeromehardaway**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,121 +5,211 @@
...
+# Week 5 SMACSS
+- Categorization
+- Naming Conventions
+- State- Based Design
+
+# Week 6 - JavaScript && jQuery && ES6.
 
 - ES3 - purpose and how to use
-- DOM manipulation with JS
+    - Types
```
Comment (kyleshevlin) [inline README.md]: I think `Types` is misleading given the proliferation of static types out there. Perhaps for the students, it's fine, but `Primitives` would be more accurate.

File: README.md [hunk 2]
```diff
@@ -5,121 +5,211 @@
...
 - ES6
-- Constructors, Objects, 'this' Keyword, Loops etc.
-- Deeper JavaScript
+    - Let and Const 
+    - Arrow Functions
+    - Default Parameters
+    - Async/Await
+    - Promises
+    - For Of Loops
+    - Spread Attributes
```
Comment (kyleshevlin) [inline README.md]: More accurate would be `Spread Operators and Rest Parameters`

File: README.md [hunk 3]
```diff
@@ -5,121 +5,211 @@
...
+-  DOM manipulation ( Again)
+-  Fetch
+- Animation and Manipulating Graphics
+- Audio And Video APIs
+- Device APIs
+- Client-side Storage APIs
+- Third Party APIs
+- Serving to The Web
+    - Github Pages
+    - Netifly
```
Comment (kyleshevlin) [inline README.md]: Typo: `Netlify`

File: README.md (hunk 2 above)
Comment (jeromehardaway) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (jeromehardaway) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (jeromehardaway) [reply README.md]: Fixed

**Pull request merged**